translations/src/config.js

```diff
@@ -68,18 +69,53 @@ class Config {
     return Object.keys(this.supportedLanguages)
   }
 
-  getRTLLanguages (): string[] {
-    return entries(this.supportedLanguages)
-      .filter(([_, languageConfig]) => languageConfig.rtl)
-      .map(([languageCode, _]) => languageCode)
+  getSupportedLanguage (languageCode: string): LanguageType | void {
+    const fallbacks = this.fallbacks[languageCode]
+
+    if (fallbacks) {
+      const found = fallbacks.find(fallback => !!this.supportedLanguages[fallback])
+
+      if (found) {
+        return this.supportedLanguages[found]
+      }
+    }
+
+    return this.supportedLanguages[languageCode]
+  }
+
+  isSupportedLanguage (languageCode: string): boolean {
+    return !!this.getSupportedLanguage(languageCode)
   }
 
-  isRTLLanguage (languageCode: string): boolean {
-    return this.getRTLLanguages().includes(languageCode)
+  /**
+   * Checks whether the languageCode belongs has an RTL script. This decision is made by the project "Integreat".
+   * Writing direction is not an attribute of "language", but of "scripts". That means that there are language which
+   * can have RTL and LTR scripts.
+   *
+   * <b>Typical Usage</b>
+   *
+   * If you do not know whether to display UI elements RTL/LTR you can use this method. Browsers for example, do not
+   * disclose the system script direction. On Android you can use the system libraries instead of this method. This only
+   * works for {@link #supportedLanguages} and not for arbitrary ones. This is because we simply do now know it and can
+   * not know it.
+   *
+   * @see http://www.i18nguy.com/temp/rtl.html
+   * @param languageCode for the check
+   * @returns {*} whether script is RTL
+   */
+  hasRTLScript (languageCode: string): boolean {
```

Comment:
I don't like the name script to be honest, but probably that's just because I am not a native speaker and haven't ever used the term script in this context.

Reply:
See the attached link. Scripts have directions not languages. I went this this name to highlight this.